test/check/events.go

```diff
@@ -0,0 +1,211 @@
+package check
```

Comment:
What is the source of this module?

Reply:
Sorry forgot to add it to the commit message. Added it now.
https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L234